glitter/blockadmin/blocks.py

```diff
@@ -36,7 +37,11 @@ def __init__(self, *args, **kwargs):
     # Use the block admin class by default
     def register(self, model_or_iterable, admin_class=None, **options):
         if not admin_class:
-            admin_class = BlockModelAdmin
+            admin_class = BlockAdmin
+
+        category = options.pop('category', None)
+        if category:
+            self.register_block(model_or_iterable, category)
```

Comment:
`register_block` can't currently take an iterable? Though that should be a nice easy one to fix.